stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -454,7 +454,10 @@ public struct Set<Element : Hashable> :
     return _variantStorage.endIndex
   }
 
-  // TODO: swift-3-indexing-model - add docs
+  /// Returns the index after the given one. 
```

Comment:
Actually, we don't need to do this anymore unless there's something you need to change about the docs that come from the protocol.